assertions-examples/src/test/java/org/assertj/examples/neo4j/PathAssertionExamples.java

```diff
@@ -45,4 +45,21 @@ public void path_assertion_examples() {
         .doesNotContainNull();
     }
   }
+
+  @Test
+  public void path_assertion_examples_introduced_in_2_0_0() throws Exception {
+    try (Transaction ignored = graphDatabase().beginTx()) {
+      // let us find the shortest path between Son Goku and Dr. Flappe
+      DragonBallGraphRepository dragonBallGraphRepository = dragonBallGraphRepository();
+      Path sonGokuToDoctorFlappe = dragonBallGraphRepository.findShortestPathBetween("Son Goku", "Dr. Flappe");
+      Node doctorGero = dragonBallGraphRepository.findUniqueCharacter("Dr. Gero");
+      Node tienShinhan = dragonBallGraphRepository.findUniqueCharacter("Tien Shinhan");
+      final Relationship masterShenTraining = dragonBallGraphRepository.findUniqueTraining("Master Shen");
+      // enjoy the negation assertions!
+      assertThat(sonGokuToDoctorFlappe)
+          .doesNotStartWith(doctorGero)
```

Comment:
this should really be `doesNotStartWithNode` and I did not spot that mistake before :(

I suggest we merge this PR anyway and update the examples with another PR when assertj-neo4j 2.0.1 is out with this small fix (I know it introduces an incompability with 2.0.0 but I do not think this is a big issue if we release it fast enough).